src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/CreateTableAttempt.java

```diff
@@ -73,8 +75,15 @@ protected SimpleStatement buildStatement() {
     // Add all primary keys and colunms
     CreateTable createTable = addColumnsAndKeys(create);
 
-    // Add comment which has table properties for vectorize
-    CreateTableWithOptions createWithOptions = createTable.withComment(comment);
+    // Add customProperties which has table properties for vectorize
+    // Convert value to ByteBuffer since extension option accepts ByteBuffer
```

Comment:
Is this comment correct? Looks like we are for some reason taking UTF-8 bytes of our payload, hex-encoding them as a String? No ByteBuffer involved as far as I can see.

I assume hex-encoding required by driver? Would be good to have a link to something that explains this -- also, if I understood earlier discussions correctly, this is temporary workaround of some kind, to be removed when driver exposes extensions as first-class concept. If so, would be good to add a note (apologies if I completely misunderstood meaning here).

Reply:
Did too many iteration to get here. Missed to change the comments. Updated it now.